src/main/scala/mesosphere/mesos/TaskGroupBuilder.scala

```diff
@@ -35,7 +34,7 @@ object TaskGroupBuilder {
   def build(
     podDefinition: PodDefinition,
     offer: mesos.Offer,
-    runningTasks: => Iterable[Task],
+    runningInstances: => Seq[Instance],
```

Comment:
could be better in a second parameter list as it would let you do:

```
build(pod, offer, someNewInstanceId, config) { blahBlah to get instances }
```